_rules/table-headers-attribute-refer-to-data-cells.md

```diff
@@ -0,0 +1,208 @@
+---
+name: `headers` attribute only refers to `cells` in the same `table` element
+
+test_type: atomic
+
+description: |
+  This rule checks that the `headers` attribute must only refer to `cells` in the same `table` element
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to `td` and `th` elements with a `headers` attribute and within a `table` element, that is either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element with the `headers` attribute refers to other `cells` of the same `table` element.
```

Comment:
I find using "refers to other cells" to be interpretable. What about something more clear like:
All `id`(s) listed in the `headers` attribute of the target element match the `id` of a `cell` in the same `table` element.

Also, the expectation leaves out the need for the cell with the matching id to be a header element. I suppose that's on purpose.

Reply:
@WilcoFiers No, I don't know any UA that has issues with it. From that perspective, I don't have a problem with the expectation leaving out the need for the cell to be a header. I guess I'm just confused because sometimes we seem to try to follow the latest specs while other times we don't :-)

Reply:
No I think  you're right @carlosapaduarte With nvda/ff for example, referencing td element in headers seems to not working, no headings are announced by the AT.

````html
 <table>
	<tr>
		<td id="one">my first header</td>
		<td id="two">second header</td>
	</tr>
	<tr>
		<td headers="two">cell</td>
		<td headers="one">other cell</td>
	</tr>
</table>
````

When focus on td `cell` or `other cell` no heading is announcing by the screen reader, that just says "cell" or "other cell".

Replacing td by th

````html
<th id="one">my first header</th>
<th id="two">second header</th>
````

and when the cell is focused, the AT announces the heading  : "my first header comun one my first header"

Reply:
Plus, even it is not required to have a complete validation, it is a code validation error not to add id  that are not present on `th` element

![image](https://user-images.githubusercontent.com/409392/56261556-beff0880-60db-11e9-99a4-8340b528cf8e.png)